rotkehlchen/exchanges/iconomi.py

```diff
@@ -0,0 +1,250 @@
+import base64
+import hashlib
+import hmac
+import logging
+import time
+from json.decoder import JSONDecodeError
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
+from urllib.parse import urlencode
+
+from requests.exceptions import ReadTimeout
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.errors import RemoteError, UnknownAsset
+from rotkehlchen.exchanges.data_structures import Location, Price, Trade, TradePair, TradeType
+from rotkehlchen.exchanges.exchange import ExchangeInterface
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.typing import ApiKey, ApiSecret, Timestamp
+from rotkehlchen.user_messages import MessagesAggregator
+from rotkehlchen.utils.serialization import rlk_jsonloads
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+SUPPORTED_FUND_TICKERS = ('BLX',)
+
+
+def iconomi_asset(asset: str) -> Asset:
+    return Asset(asset.upper())
+
+
+def iconomi_pair_to_world(pair: str) -> Tuple[Asset, Asset]:
+    tx_asset = iconomi_asset(pair[:3])
+    native_asset = iconomi_asset(pair[3:])
+    return tx_asset, native_asset
+
+
+def trade_from_iconomi(raw_trade: Dict) -> Trade:
+
+    timestamp = raw_trade['timestamp']
+
+    if raw_trade['type'] == 'buy_fund' \
```

Comment:
can you please change this? In the code we avoid splitting lines with `\`

Instead do it like below:

```python
unknown asset = (
    raw_trade['type'] == 'buy_fund' and
    raw_trade['target_ticker'] not in SUPPORTED_FUND_TICKERS
)
if unknown_asset:
    ...
```

Both here and in the rest of the places where you split the line like this below.

Reply:
Is this ok?

```
if (                                                                                                         
    raw_trade['type'] == 'buy_fund' and                                                                      
    raw_trade['target_ticker'] not in SUPPORTED_FUND_TICKERS                                                 
):                                                                                                           
    raise UnknownAsset(raw_trade['target_ticker'])
```

Reply:
yeah should be. The only reason for the extra variable is readability.